test/quantization/test_equalize.py

```diff
@@ -0,0 +1,95 @@
+import torch
+import torch.nn as nn
+
+from torch.testing._internal.common_quantization import QuantizationTestCase
+
+import torch.quantization._equalize as _equalize
+
+import copy
+
+class TestEqualizeEager(QuantizationTestCase):
+    def checkChannelsEqualized(self, tensor1, tensor2, output_axis, input_axis):
+        ''' Checks the channel ranges of tensor1, tensor2 are the same,
+        which is an indication that equalization has been applied correctly
+        '''
+        output_channel_tensor1 = _equalize.channel_range(tensor1, output_axis)
+        input_channel_tensor2 = _equalize.channel_range(tensor2, input_axis)
+
+        # ensuring the channels ranges of tensor1's input is the same as
+        # tensor2's output
+        self.assertEqual(output_channel_tensor1, input_channel_tensor2)
+
+    def getModule(self, model, name):
+        ''' Given the name is a submodule to a model, return the submodule
+        '''
+        curr = model
+        name = name.split('.')
+        for subname in name:
+            curr = curr._modules[subname]
+        return curr
```

Comment:
nit: I think we can put this into external utility file. But this doesn't have to be part of this PR